functional-tests/entry.py

```diff
@@ -7,13 +7,46 @@
 from typing import Optional
 
 import flexitest
+from strata_utils import get_address
```

Comment:
There's also the `get_change_address` but I think we'll be fine for now with just `get_address`.